src/search-params.ts

```diff
@@ -36,4 +43,6 @@ export interface SearchParams {
   fields?: string[];
 
   aggregations?: AggregateSearchParams;
+
+  aggregationsSize?: number;
```

Comment:
I can ask Ximm about this, but I don't think advanced aggregations objects (with per-field sizes) are currently supported by the PPS -- only simple comma-delimited aggregations. So the `aggregationsSize` parameter is an alternative way to set the size across all aggregation types.

This means we don't have the flexibility to, say, fetch 2 `subject` buckets and 4 `mediatype` buckets in the same request, since we only have a single size parameter. I'm not sure whether we have a particular use case that requires a greater level of granularity than that?